ingestion/src/metadata/ingestion/source/databricks.py

```diff
@@ -19,6 +26,69 @@
 from metadata.ingestion.source.sql_source_common import SQLConnectionConfig
 
 
+class STRUCT(String):
+    """The SQL STRUCT type."""
+
+    __visit_name__ = "STRUCT"
+
+
+class ARRAY(String):
+    """The SQL ARRAY type."""
+
+    __visit_name__ = "ARRAY"
+
+
+class MAP(String):
+    """The SQL MAP type."""
+
+    __visit_name__ = "MAP"
+
+
+_type_map.update({"struct": STRUCT, "array": ARRAY, "map": MAP})
+
+
+@reflection.cache
+def get_columns(self, connection, table_name, schema=None, **kw):
+    # override to get columns properly; the reason is because databricks
+    # presents the partition information differently from oss hive
+    rows = self._get_table_columns(connection, table_name, schema)
+    # Strip whitespace
+    rows = [[col.strip() if col else None for col in row] for row in rows]
+    # Filter out empty rows and comment
+    rows = [row for row in rows if row[0] and row[0] != "# col_name"]
```

Comment:
If possible, let's try to add different names to variables with different purposes instead of re-assigning. This helps the reader better understand the intent